moveit_ros/planning_interface/package.xml

```diff
@@ -43,6 +43,9 @@
   <test_depend>moveit_resources_panda_moveit_config</test_depend>
   <test_depend>moveit_simple_controller_manager</test_depend>
   <test_depend>moveit_planners_ompl</test_depend>
+  <test_depend>moveit_planners_chomp</test_depend>
+  <test_depend>moveit_planners_stomp</test_depend>
+  <test_depend>pilz_industrial_motion_planner</test_depend>
```

Comment:
so this adds a circular test dependency, even though the planning interface is not listed here. I think we should have a higher-level package for running integration tests for that purpose.

Reply:
I moved the test into its own package. At the moment it is part of moveit_ros but we can move it somewhere else too